db/migrate/20180312163100-create-score.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('scores', {
+      uri: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING
+      },
+      unaware: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1
+      },
+      curious: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1
+      },
+      follower: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1
+      },
+      guide: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1
+      },
+      confidence: {
+        type: Sequelize.DOUBLE,
+        defaultValue: 0
+      }
```

Comment:
I believe `created_at` and `updated_at` need to be defined in the migration unless `timestamps: false` is defined in the model options